mcs/class/System.IO.Compression/SharpCompress/IO/ReadOnlySubStream.cs

```diff
@@ -27,6 +28,8 @@ protected override void Dispose(bool disposing)
             }
         }
 
+        private long length { get; set; }
```

Comment:
I couldn't find this pattern in the existing code and since this is the backing store for the `Length` property making this a field feels more consistent with usual C# conventions :)